### DISCUSSION TITLE
Hide prereleases from activity

### DISCUSSION THREAD

**Discussion by mgreenw** [community/community#119672] [Other Feature Feedback, Questions, & Ideas]
Hello! The activity section of the explore tab is cluttered with prereleases of repos that I follow. Would it be possible to hide prereleases by default, and add a setting to show them? The section is basically useless because I can only see things like 5.7.12-alpha.10, which I really don't care about day to day. Thanks you!

**Comment by LiteBrite82** [comment]
Hi @mgreenw 

Thanks so much for your community participation and feedback! 

This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
